nav_admin.php

```diff
@@ -0,0 +1,21 @@
+<!--nav for admin page -->
```

Comment:
2/9/2021: I also create a admin nav page so in the php page just need to called it. Admin page in include setting which admin can access to some special command(eg delete user)